class-two-factor-core.php

```diff
@@ -897,6 +917,7 @@ public static function login_form_validate_2fa() {
 			$rememberme = true;
 		}
 
+		remove_filter( 'send_auth_cookies', '__return_false', PHP_INT_MAX );
```

Comment:
It's worth noting that this is a just-in-case filter removal.
The filter should not normally be set as far as I can tell, but I figured it was better to remove it than potentially break an edge-case in an untested provider which ran the `authenticate` filter during it's validation process.

Reply:
That sounds wise 👍🏻 

IMO it'd be good to copy/paste that into an inline comment, so folks who are curious in the future won't have to go looking through `git blame` for this PR.